cartofacade/capi_test.go

```diff
@@ -12,6 +12,47 @@ import (
 	"go.viam.com/utils/artifact"
 )
 
+func positionIsZero(t *testing.T, position GetPosition) {
+	test.That(t, position.X, test.ShouldEqual, 0)
+	test.That(t, position.Y, test.ShouldEqual, 0)
+	test.That(t, position.Z, test.ShouldEqual, 0)
+
+	test.That(t, position.Imag, test.ShouldEqual, 0)
+	test.That(t, position.Jmag, test.ShouldEqual, 0)
+	test.That(t, position.Kmag, test.ShouldEqual, 0)
+	test.That(t, position.Real, test.ShouldEqual, 1)
```

Comment:
This should indeed be 1.

This is the default initialized state of the `cartographer::transform::Rigid3d` struct.

I would rather have tests that show specificity when it doesn't interfere with having reliable tests. This passes on all platforms and as a result I think testing that it is non 0 gives up test clarity for no bennefit.

Reply:
Got it, agree that if it's initialized to 1, we should check 1 rather than non-0 here.

Does that mean in the other place we should check non-0 and non-1?

Reply:
good catch. updated